src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -12,7 +14,25 @@
     <t:base/>
     <div class="gcse-searchbox-only" data-resultsUrl="/search"></div>
     <div class="srp-container">
-      <div class="gcse-searchresults-only"></div>
+      <div class="srp-child">
+        <div class="gcse-searchresults-only"></div>
+      </div>
+      <div class="srp-child">
+        <%ArrayList<Card> cards = (ArrayList<Card>)request.getAttribute("card_list");%>
+          <div class="card">
+            <div class="card-content">
+              <a href="<%=cards.get(0).getLink()%>" title="card-source" target="_blank">
```

Comment:
Most people expect the page to navigate to the search result on click, not open a new tab.